### PR TITLE
Api data build

### PR DESCRIPTION
**Pull request by bhohnco**
### Is this a fix or a feature?
-Feature

### What is the change or fix?
-API fetches have been added and some state changes in component files. 

### How should this be tested?
-Try to have data pulled over in the console 

### Linked Issues:
- #7

#### Add A Reviewer if Necessary
Added

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-redux": "^7.2.4",
+    "react-router-dom": "^5.2.0",
```
Comment (pcmueller) [inline package.json:12]: Good call adding Router, I'll install after pulling down

File: src/components/App/App.css [hunk 2]
```diff
@@ -3,6 +3,13 @@
   padding: 0;
 }
 
+
```
Comment (pcmueller) [inline src/components/App/App.css:6]: Looks like the conflict is that I deleted all the old App.css styling (now that the logos are gone)

File: src/components/App/App.js [hunk 3]
```diff
@@ -8,13 +9,39 @@ import './App.css';
...
+      topTracks: [],
+      selectedArtist: null,
+      selectedArtistImage: null,
+      input: "",
+      error: ""
     }
+    console.log(this.state.topArtists)
   }
 
+  componentDidMount = async () => {
```
Comment (pcmueller) [inline src/components/App/App.js:23]: Nice work on all the API calls

**Review by pcmueller — COMMENTED** [review]
Thanks for all of this, Bryan

File: src/components/App/App.js [hunk 4]
```diff
@@ -8,13 +9,39 @@ import './App.css';
...
   constructor() {
     super();
-    this.state={
-      nation: '',
-      topArtists: '',
-      topTracks: ''
+    this.state = {
+      topArtists: [],
+      topTracks: [],
+      selectedArtist: null,
```
Comment (battan40) [inline src/components/App/App.js:15]: Maybe lets consider using truthy here.

File: src/utilities/apiCalls.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+      .then(response => {
+        if (response.ok) {
+          return console.log(response.json())
+        } else {
+          throw new Error(`Error, please try again!`)
+        }
+      })
+}
+
+// export const getSelectedArtist = (id) => {
```
Comment (battan40) [inline src/utilities/apiCalls.js:33]: Lets make sure we don't push up commented out code, when possible.

File: src/utilities/apiCalls.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+export const getTopTracks = () => {
+  console.log("topTracks")
+  return fetch(topTracksByCountryUrl)
+      .then(response => {
+        if (response.ok) {
+          return console.log(response.json())
+        } else {
+          throw new Error(`Error, please try again!`)
```
Comment (battan40) [inline src/utilities/apiCalls.js:28]: Thank you for getting error handling going.

**Review by battan40 — APPROVED** [review]

**Pull request merged**